device-connectors/src/testflinger_device_connectors/devices/zapper_kvm/README.md

```diff
@@ -46,3 +48,13 @@ The tool will select the storage device with the following priority:
 ### Noble
 
 Ubuntu OEM 24.04 uses `autoinstall`. The procedure and the arguments are the same as _vanilla_ Ubuntu.
+
+## Live ISO
+
+Support for live ISOs is simply performed booting from an external storage device and returning.
+
+### Job parameters
+
+- __boot_from_ext_media__: should be set to "true"
+- __wait_until_ssh__: SSH assertion at the end of provisioning can be skipped, in case the live ISO does not include an SSH server
```

Comment:
The Zapper at the end of provisioning usually waits until SSH is available. In case of a live ISO, it's possible that the SSH server is not available: that would make the Zapper `provision` API fail. If true, at the end of provisioning the Zapper doesn't wait for SSH and returns. I'll add a note saying this is happening on the Zapper side.

Reply:
If setting this to true means "skip the step on the zapper that waits for ssh", then should we just calle it "skip_wait_for_ssh"?

Another option to make it less confusing though, would be to invert the logic, but still call it "wait_for_ssh", but if it's true, then that means we "wait for ssh to be available", and if it's false, then we don't.

Reply:
Sorry for the confusion, the comment is wrong. True means Zapper waits for SSH, False means it skips the SSH test. Just updated the comment.

Reply:
That sounds much better, thanks!